src/harness/loggedIO.ts

```diff
@@ -14,19 +14,19 @@ interface FileInformation {
 interface FindFileResult {
 }
 
-interface IOLogFile {
+interface IoLogFile {
     path: string;
     codepage: number;
     result?: FileInformation;
 }
 
-interface IOLog {
+interface IoLog {

```

Comment:
Io is a rock orbiting Jupiter, here it's IO -  common JavaScript convention is to keep abbreviations as ALLCAPS unlike C#. Compare System.Xml and window.URL.